hack/ci/push-latest-cli/push-latest-cli.sh

```diff
@@ -14,18 +14,25 @@
 # limitations under the License.
 
 set -o errexit -o nounset -o pipefail
-set -x;
+set -x
+
+# Function: Upload file to GCS bucket
+upload_file() {
+    local file="$1"
+    local version="$2"
+    gsutil -m cp -P "${file}" "gs://${BUCKET}/${version}/$(basename "${file}")"
```

Comment:
`-m` doesn't make sense for small, single file interactions?

And otherwise this was just `gsutil cp -P <call site specific arguments>` which is clearer inline, similar to `cp -R ...`

This seems over-abstracted